pedalboard/RubberbandPlugin.h

```diff
@@ -76,10 +113,23 @@ Base class for rubberband plugins.
     }
 
     // Pull the next audio data out of Rubberband:
-    rbPtr->retrieve(outBlock, samplesToPull);
+    return rbPtr->retrieve(outBlock, samplesToPull);
   }
 
 protected:
+  virtual int getLatencyHint() override {
+    if (!rbPtr)
+      return 0;
+
+    initialSamplesRequired =
+        std::max(initialSamplesRequired,
+                 (int)(rbPtr->getSamplesRequired() + rbPtr->getLatency() +
+                       lastSpec.maximumBlockSize));
```

Comment:
Why add the blockSize as well? The documentation doesn't seem to suggest it's related to latency.

Reply:
Looks like we're conflating the latency and samples required for processing based on Chris' answer. Will it be practical to implement the suggestion of gathering more samples for the first process?